opentelekomcloud/services/ecs/compute_instance_v2_networking.go

```diff
@@ -423,24 +420,25 @@ func FlattenInstanceNetworks(
 					"mac":         instanceNIC.MAC,
 				}
 
-				// Use the same method as getAllInstanceNetworks to get the network uuid
-				networkInfo, err := GetInstanceNetworkInfo(d, meta, "name", instanceAddresses.NetworkName)
+				serverNICs, err := servers.GetNICs(computeClient, server.ID).Extract()
 				if err != nil {
-					log.Printf("[WARN] Error getting default network uuid: %s", err)
-				} else {
-					if v["uuid"] != nil {
-						v["uuid"] = networkInfo["uuid"].(string)
-					} else {
-						log.Printf("[WARN] Could not get default network uuid")
+					return nil, err
+				}
```

Comment:
This API call should be done out of all cycles

Reply:
Done